### PR TITLE
Update UG

### LINKED ISSUE #150: Complete UG

### PR DESCRIPTION
**Pull request by 3CCLY**
Fixes #150

### CODE REVIEW

**Review by Sukkaito — APPROVED** [review]
Should be good to go! I counted for number of rows in the summary table and the listing and `help` command, they are the same! Good work! 👍

**Comment by KSanjith** [comment]
LGTM

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -189,7 +189,7 @@ to finally printing the "recipe added" message to the User on the CLI:
 
 2. `EditCommand` Class
 
-- The EditCommand class finds an existing recipe in recipeList by name, and then replaces its parameter
+- The EditCommand class finds an existing recipe in RecipeList by name, and then replaces its parameter
```
Comment (seanngja) [inline docs/DeveloperGuide.md:192]: Nice catch!

**Review by seanngja — APPROVED** [review]
LGTM

**Pull request merged**